### ISSUE TITLE
[Webcompat reporting tool] [Accessibility-Voice Over] - "Optional" is not announced

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#35324]
### Steps to reproduce

1. Fresh install FF.
2. Access a webpage then tap on 3 dot menu.
3. Expand the menu then tap on the Report Broken Site button.
4. Enable voice over and go to the description field.
5. Listen.

### Expected behavior

“Optional” is announced with voice over.

### Actual behavior

“Optional” is not announced with voice over.

### Device & build information

- Device: iPhone 16  (iOS 26.3)
- Build version: v153.3 (75262)
- First seen version: N/A

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-16495)

**Comment by data-sync-user** [comment]
> **Diana Andreea Barladeanu** commented from Jira:

The issue is still reproducible. Validated in v155.0 (76235), with iPhone 15 (17.6).

**Comment by data-sync-user** [comment]
> **Razvan Litianu** commented from Jira:

hello `@Diana Andreea Barladeanu`, this should be in 155.1, sorry for the confusion.
